src/configure/clients/github/TemplateServiceClient.ts

```diff
@@ -1,19 +1,505 @@
+import { ExtendedPipelineTemplate } from "../../model/Contracts";
 import { RepositoryAnalysisParameters } from "../../model/models";
 import { PipelineTemplateMetadata } from "../../model/templateModels";
 import { RestClient } from "../restClient";
 
 export class TemplateServiceClient {
     private restClient: RestClient;
+    private  readonly templateServiceUri: string = "https://tswithouthmac.azurewebsites.net/Templates";
 
     public constructor() {
         this.restClient = new RestClient();
     }
 
     public async getTemplates(body: RepositoryAnalysisParameters): Promise<PipelineTemplateMetadata[]> {
         return this.restClient.sendRequest2(
-            'https://ts21.azurewebsites.net/Templates',
+            this.templateServiceUri,
             'POST',
             '2019-05-01',
```

Comment:
this is interesting! I did now know that we have versions in template service currently. and that too in this format. @tejasd1990 we should discuss about the version and its format

Reply:
I don't think that API has a version.